### DISCUSSION TITLE
how to remove attributes otel.library.version and otel.library.name from span

### DISCUSSION THREAD

**Discussion by kadhamecha-conga** [open-telemetry/opentelemetry-collector-contrib#29004] [General]
hi team,

i want to remove attributes such as otel.library.version and otel.library.name from spans.

i tried attributes processor. but it is not removing the attributes. 

can someone please help me to remove these attributes ?

thanks.

**Comment by ovidiubuligan** [comment]
you will have to use the resource processor  since  those are resource attributes of the span : https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourceprocessor/README.md 

It removes `telemetry.sdk.language` and similar ,but those specific attributes  don't get removed, I don't know why...
